components/Navigation/navigation.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Header's</title>
+    <link rel="stylesheet" href="navigation.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    
+    <div class="box">
+        
+        <div class="header">
+            <div class="headertxt">
+                <div class="btn"> <button><i><i class="fa fa-bars"></i></i></button><span>Hi, Gravitor!</span></div>
+           
+        </div>
+        
+        <div class="actionbtn">
+            <div class="btn"><button><i class= "fa fa-download">	</i></button>
+                <button><i class="fa fa-print">	</i></button>
+                <button><i class="fa fa-bookmark">	</i></button></div>
+            
+        
+        </div>
+        </div>
+        <div class="header1">
+            <div class="headertxt1">
+                <div class="btn1"> <button><i><i class="fa fa-bars"></i></i></button><span>Hi, Gravitor!</span></div>
+           
+        </div>
+        
+        <div class="actionbtn1">
+            <div class="btn"><button><i class= "fa fa-download">	</i></button>
+                <button><i class="fa fa-print">	</i></button>
+                <button><i class="fa fa-bookmark">	</i></button></div>
+            
+        
+        </div>
+        </div>
+        
+    
```

Comment:
Indentation missing. Please format the code

Reply:
dsone